### ISSUE TITLE
[RELEASED_PREQUAL_COMPLETE] Rada survivor → expanded V9 → retained inventory/decontam cross-bind

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1775]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SCIENTIFIC_CONTROL: #548
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
STATUS: ACTIVE_NONOWNING_PREQUAL_PENDING_POSTCLAIM_DIRECT_ARBITRATION
LOCAL_FREE_ONLY: true
PRODUCT_MUTATION: false
AUTHORITATIVE_PR_AUDIT_VERDICT: false
MERGE_AUTHORITY: false
TRAINING_CLAIM: false

SWARM_LANE_KEY: PREQUAL|PR1017-PR1459-PR945|RADA-SURVIVOR-TO-GLOBAL-DEDUP-TO-BALANCE|AUTHORITY-AND-CAPACITY-INTAKE-CROSSBIND|V1
SELECTED_WORK_KIND: NONOWNING CROSS-BOUNDARY INDEPENDENT_VERIFY / CRITICAL-PATH DATA AUTHORITY COMPOSITION
LIVE_MAIN_AT_CLAIM: 2da11c62f066671a1122888ec116242502f4fb9c
UPSTREAM_RADA_PR: #1017 @ a54ae30943ba3f25a2c1e7e5bf6b42c5ca89a1a3
UPSTREAM_RADA_AUDITOR: #1771 (active; this package does NOT duplicate that audit)
INDEXED_EXECUTOR_PR: #1459 @ 2fddd8fbc46dc7c878f0fafc1da3046283e91cc0
INDEXED_EXECUTOR_OWNER: #1764 / lawful successor (this package does NOT mutate/audit PR1459)
BALANCE_PR: #945 @ eddc498cc330b1e4b255eb4e5712f13391dd8169
BALANCE_STATE: fresh different-worker PASS already reported by #1756; this package does NOT re-audit/mutate PR945
EXPANDED_V9_EXECUTION: #1768 / PR1491 (active; this package does NOT duplicate execution)

WHY THIS UNIT NOW:
- The binding scientific launch state remains AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0 and TOKENIZER_FIT_AUTHORIZED=false.
- Real post-G05/G06 materialization exists but is only 255 records / 5,430,017 retained UTF-8 payload bytes on the current physical path.
- The dominant near-term supply is the physically executed Rada no-site replay: 4,619 survivors / 858,698,060 UTF-8 bytes, still correctly zero-credit pending exact PR1017 audit/integration and all downstream gates.
- The exact producer audit (#1771), indexed global-dedup Product repair (#1764), real expanded-V9 execution (#1768) and balance mechanics are separately owned or terminalized. Duplicating them would increase collision risk.
- The remaining independent binary question is whether the existing authority/data shapes can actually compose, without caller-authored trust or silent payload loss, from the PR1017 survivor evidence into incumbent global-dedup execution and then into the already-repaired post-materialization balance selector. Closing this now can prevent the current owners from reaching terminal states that still cannot feed each other.

BINARY QUESTIONS AFTER ARBITRATION WIN:
1. Does the current Rada terminal evidence + existing authenticated launcher expose every independently fixed identity/count/byte/root needed by the incumbent expanded/global-dedup intake, or does a downstream adapter still need an authority-bearing field that PR1017 does not persist?
2. Does the global-dedup result contract carry enough exact survivor/family/provenance/payload authority for PR945's repaired current physical/family-vector contract, or is there an unbound bridge where a caller could construct/reseal the balance input from prose/derived data?
3. If interfaces are composition-complete, what is the exact shortest zero-credit execution order/identity chain that active owners should use so the ~858.7MB Rada survivor can become measurable post-dedup/post-balance capacity without inventing architecture #2?

OWNED DIAGNOSTIC SURFACE ONLY:
- read-only exact-code/tree/diff inspection of PR1017@a54ae309, PR1459@2fddd8fb, PR945@eddc498c, incumbent merged expanded-V9/current-main interfaces, and exact relevant execution/evidence schemas;
- trace source/survivor identities, record inventories, payload hashes/bytes, family/provenance identities, zero-credit flags, and externally expected trust roots across the existing call/data path;
- adversarial source-level reasoning for omission/substitution/coherent reseal/candidate-derived expected identities; execute only cheap focused repository-independent probes if they do not duplicate active CI or real payload runs;
- publish one concrete HIGH/MEDIUM cross-boundary finding with a minimum SAME existing-lineage repair contract, or source-backed composition closure with the exact next execution order;
- hand result to #1771/#1764/#1768, PR945 lawful successor and #1760/lawful integration successor. No branch write, PR write, merge or real payload replay under this claim.

STRICT NONOVERLAP:
- no PR1017 exact-head audit verdict; #1771 owns it;
- no PR1459 Product repair/audit; #1764 owns the current repair lineage;
- no PR1491 expanded-V9 execution; #1768 owns it;
- no PR945 Product/audit mutation; its current head already has fresh PASS from #1756;
- no central integration, tokenizer/packing/unique-loss/readiness/runtime/recovery/training/evaluation/final-test mutation;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence.

PACKAGE DIMENSIONS:
- independent verification / exact source-interface analysis
- adversarial trust-direction and substitution analysis
- live authority binding
- end-to-end data/authority composition proof
- reproducibility/identity closure
- operator/owner repair-or-execution handoff

ACCEPTANCE:
1. Mandatory postclaim direct OPEN issue collection newest-first at 100/page reaches exact empty sentinel; separate OPEN PR collection reaches exact empty sentinel. Any earlier valid same-semantic owner wins; this worker yields before authoritative finding.
2. Immediately before verdict rebind live main, target heads/states and active owners. Any head movement invalidates conclusions for that target until rebound.
3. Identify concrete schemas/functions/fields at each boundary. Prose compatibility is insufficient.
4. Every downstream consumed identity must be externally rooted or mechanically derived from an authenticated upstream object with no candidate-derived expected root; self-consistent reseal under unchanged trust must fail closed.
5. Missing/extra/substituted records, survivor payload hash/byte drift, family/provenance reclassification and zero-credit widening must reject before downstream capacity promotion.
6. Closure requires either a direct current-interface proof plus exact execution order, or one minimal patch-ready same-lineage repair contract; do not create a second dedup/balance framework.
7. This prequal grants zero canonical corpus/training credit and never upgrades scientific truth by itself.

SCIENTIFIC_TRUTH_AT_CLAIM:
REAL_RADA_NO_SITE_REPLAY_EXECUTED=true
CANONICAL_RADA_SURVIVOR_AUTHORITY=false pending #1771 audit/integration
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_UNIQUE_LOSS_POSITIONS=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT NOW:
Complete mandatory direct OPEN issues + OPEN PR arbitration to exact empty sentinels, final race-refresh this semantic and exact heads, then inspect the real schemas/interfaces and publish one concrete closure/finding.

**Comment by Oleksii-debug** [comment]
STATUS: RELEASED_PREQUAL_COMPLETE
VERDICT: COMPOSITION_PATH_IDENTIFIED / NO_SECOND_FRAMEWORK_NEEDED
PRODUCT_MUTATION: none
AUTHORITATIVE_PR_AUDIT_VERDICT: none

POSTCLAIM ARBITRATION PROOF:
- OPEN issues newest-first: pages 1..12 non-empty, page 13 exact `[]` sentinel.
- OPEN PRs newest-first: pages 1..5 non-empty, page 6 exact `[]` sentinel.
- final exact semantic refresh found no earlier valid owner for this read-only cross-bind surface.

SOURCE-BACKED RESULT:
1. PR1459 is **not** on the PR1017 Rada_Trees path. It is the separate Rada-laws indexed-dedup scale-mechanics lineage. Treating it as a prerequisite here would add a false dependency.
2. Actual current lineage is:
   `PR1017 terminal Rada survivor → PR1491 merged expanded-V9 execution → PR940 retained-inventory freeze → current generic reserved-eval decontamination → G05/G06 coverage/materialization → PR945 repaired balance → split/pack/unique-loss`.
3. PR940 exact head `109a2933cd4a9c9c62906acb8668423d531f3490` is interface-complete for merged V9 `report.json` + `survivors.json`. Freeze is text-free and externally identity-bound. `prepare_ephemeral_data232_rows()` independently binds the retained-inventory identity and rejects missing/extra payload rows or normalized/comparison payload hash/byte drift before emitting ephemeral matcher rows. No second retained-inventory architecture is needed.
4. PR940 Product state is already terminalized/released by #1557: exact-head CI runs `34790399832=SUCCESS` and `34776411681=SUCCESS`; focused 9 tests PASS; full suite 1297 PASS; current-main importability PASS; PR remained OPEN/non-draft/cleanly mergeable at that qualification. I routed this exact existing intake to current integration owner #1760 in comment `5657410894`.
5. The old `tools/run_data526_v8_reserved_decontamination.py` is DATA-526-specific, but merged `src/twelve_six/data/current_reserved_decontamination_v1.py` is already generic over `12-6.d03.postdedup-decontam-handoff.v1`. Therefore the next downstream operational seam is execution/orchestration: reconstruct exact expanded survivor payload rows, call PR940 `prepare_ephemeral_data232_rows()`, then feed the existing current generic decontamination authority with independently fixed expected identities. This is an execution bridge, not a new matcher/decontam algorithm.

LIVE UPSTREAM UPDATE DURING THIS PREQUAL:
- #1771 independently terminal-PASSed PR1017 exact cleanup head `a54ae30943ba3f25a2c1e7e5bf6b42c5ca89a1a3`; physical artifact `10310591743` = 4,619 unique survivors / 858,698,060 UTF-8 bytes, survivor SHA `9c76085e47b514ab0a49ba8ccf0b51f692859b60360275b4dd0eb5e9d78305a8`.
- PR1491 first execution attempt run `34792511437` failed before V9 execution because the carrier assumed Rada survivor/report members were top-level; sealed artifact stores them under exact `authoritative-a/`. That failure was already independently triaged to #1768, so this worker did not duplicate the comment or branch mutation.
- #1768 has now advanced PR1491 to `fc966c47a4e8abbbb9ec64e255e0cd788ef3bf0f`, binding exact `authoritative-a/`; new CI run `34792807498` is queued as of final refresh. This package does not claim its outcome.

SCIENTIFIC TRUTH UNCHANGED:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

OWNERSHIP: RELEASED. Any downstream worker must re-late-bind PR1491/PR940/current integration ownership before acting. This finding authorizes no corpus/training credit by itself.
